pysb/core.py

```diff
@@ -154,8 +154,9 @@ def rename(obj, new_name):
 
 
 class Symbol(sympy.Dummy):
-    def __new__(cls, name):
-        return super(Symbol, cls).__new__(cls, name)
+    def __new__(cls, name, **kwargs):
+        real = kwargs.pop('real', True)
```

Comment:
I think adding `real=True` to the `__new__` args instead would be cleaner.

Reply:
done